.bazelci/presubmit.yml

```diff
@@ -41,6 +41,7 @@ tasks:
     # Regenerate the BUILD file for the test module using Gazelle.
     # Also verify -repo_config are generated correctly in gazelle.bash
     - rm pkg/BUILD.bazel
+    - touch pkg/BUILD.bazel # ensure empty pkg/BUILD.bazel exists
```

Comment:
Unless we refactor `tests/bcr` more significantly, this may be required as the existing of `pkg/BUILD.bazel` is required given that go.work "points" to the `pkg` package.

That being said, if there is an alternative, I'm open to suggestions.